Array_method_lab.js

```diff
@@ -0,0 +1,265 @@
+console.log(' ');
+console.log('Question 1');
+console.log(' ');
+
+//1. Given an array of numbers, write a function that returns a new array where every element is incremented by 10.
+
+let array = [1, 2, 3, 4, 5];
+
+let newArray = array.map((el) => {
+  return el + 10;
+});
+
+console.log(array);
+console.log(newArray);
+
+console.log(' ');
+console.log('Question 2');
+console.log(' ');
+
+//2. Given an array, return a new array where every non-string element is converted to an empty string.
+
+let mixedArray = ['a', 123, 'b', 'c', { name: 'cat' }];
+
+const filteredArray = (array) => {
+  return array.map(element => {
+    if (typeof element !== typeof 'string') {
+      return ' ';
+    } else {
+      return element;
+    }
+  });
+};
+
+console.log(filteredArray(mixedArray));
+
+console.log(' ');
+console.log('Question 3');
+console.log(' ');
+
+//3. Write a function that returns only the even elements in an array of numbers.
+
+let listedArray = [1, 2, 3, 4, 5];
+
+const onlyEvenArray = (array) => {
+  return array.filter(elem => {
+    return elem % 2 === 0;
+  });
+};
+
+console.log(onlyEvenArray(listedArray));
+
+console.log(' ');
+console.log('Question 4');
+console.log(' ');
+
+//4. Write a function that checks if all elements in the array are the same.
+
+let randomArray = [1, 1, 1, 1, 1];
+let randomArray2 = [1, 1, 1, 1, 2];
+// let arrayElemSame =
+
+const arrayElemSame = (array) => {
+  let pervElem = array[0];
+  return array.every((elem)  => {
+    if (elem === pervElem) {
+      return true;
+    } else {
+      return false;
+    }
+  });
+};
+
```

Comment:
Great job . Remember the every method returns a boolean so you don't need those returns. You could just return your condition.